portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSet.testcase

```diff
@@ -0,0 +1,69 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-data-set-taglib,frontend-data-set-sample-web";
```

Comment:
maybe this `frontend-data-set-taglib` module is causing issues? Is it needed? :thinking: @mateusralv

Reply:
let's try removing that and running ci:test:frontend:nocompile again

Reply:
I put it because I found it in the documentation where it explained that it was necessary to deploy the two pages, which apparently is not necessary. But now it's [passed](https://testray.liferay.com/home/-/testray/case_results/1649952341?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D1649951386%26testrayRunId%3D1649952311%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc) Thanks Susan.